psycopg/psycopgmodule.c

```diff
@@ -70,24 +70,104 @@ HIDDEN PyObject *psyco_null = NULL;
 /* The type of the cursor.description items */
 HIDDEN PyObject *psyco_DescriptionType = NULL;
 
+
+/* finds a keyword or positional arg (pops it from kwargs if found there) */
+static PyObject *
+parse_arg(int pos, char *name, PyObject *defval, PyObject *args, PyObject *kwargs)
+{
+    Py_ssize_t nargs = PyTuple_GET_SIZE(args);
```

Comment:
It wouldn't hurt to do so, but I'm not sure that would be a win.  There is only one call site now, but I think it makes it easier to reuse this function if you don't require to pass `nargs` separately.  The calculation itself is cheap of course.